### PR TITLE
ELEMENTS-797: allow using enrichers object in document

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: nuxeo-resource.html [hunk 1]
```diff
@@ -245,9 +248,19 @@
...
+            } else {
+              enrichers = this.enrichers;
+            }
+            Object.entries(enrichers).forEach(([type, value]) => {
+              if (Array.isArray(value)) {
+                value = value.join(',');
+              }
+              options.headers[`enrichers-${type}`] = value;
+            });
+
```
Comment (Gabez0r) [inline nuxeo-resource.html]: Shouldn't these changes be applied to `nuxeo-operation` as well?

File: nuxeo-document.html [hunk 2]
```diff
@@ -138,8 +138,8 @@
 
             /** The `content enricher` of the resource. */
             enrichers: {
-              type: String,
-              value: '',
+              type: Object,
+              value: {},
```
Comment (Gabez0r) [inline nuxeo-document.html]: Shouldn't these changes be applied to `nuxeo-audit-page-provider.html` and `nuxeo-page-provider.html` as well?

**Comment by Gabez0r** [comment]
👍

**Comment by guirenard** [comment]
:+1:

**Pull request merged**